sencha-workspace/packages/slate-cbl/src/view/AttachmentsList.js

```diff
@@ -82,31 +82,46 @@ Ext.define('Slate.cbl.view.AttachmentsList', {
                 action = btn.getAttribute('action');
             }
 
+            record = me.getRecord(Ext.fly(event.target).parent(me.getItemSelector()));
+
             switch (action) {
                 case 'settings':
+
                     Ext.create('Ext.menu.Menu', {
                         defaults: {
-                            xtype: 'menucheckitem'
+                            xtype: 'radiogroup',
+                            record: record,
+                            name: 'Share',
+                            columns: 1,
+                            vertical: true,
+                            handler: function(item) {
+                                item.record.set(item.name, item.inputValue);
+                            }
                         },
-                        items: [
-                            {
-                                text: 'View Only',
-                                checked: true
+                        items: [{
+                            items: [{
+                                boxLabel: 'View Only',
+                                inputValue: 'view-only',
+                                checked: Boolean(record.get('Share') == 'view-only')
```

Comment:
@nbey did some linter make you use this explicit cast? `==` should always return boolean inherently